modules/portworx/examples/simple/main.tf

```
@@ -0,0 +1,44 @@
+# TEMPLATE: This file was automatically generated with `generate_addon_structure.sh`
+# TEMPLATE: and should be modified as necessary
+# TEMPLATE:
+# TEMPLATE: All ".tf" files are parsed at once. There is no benefit to numerically prefixed filenames. Keep all resource definitions in "main.tf".
+# TEMPLATE:
+# TEMPLATE: When main.tf becomes unwieldy, consider submodules (https://www.terraform.io/docs/language/modules/develop/structure.html) 
+# TEMPLATE: and dependency inversion (https://www.terraform.io/docs/language/modules/develop/composition.html).
+#
+provider "equinix" {
+  auth_token = var.metal_auth_token
+}
+
+module "equinix_kubernetes_addons" {
+  source = "../../../../"
+
+  equinix_project = var.metal_project_id
+  equinix_metro   = "SV"
+
+  ssh_host        = var.host
+  ssh_private_key = file(var.private_key_path)
+
+  kubeconfig_remote_path = var.kubeconfig_remote_path
```

This addon uses a local installation with `kubeconfig_local_path`. We know it's unclear when to use local/remote, so we'll update the project to make it clear and simple with a single option in all addons settings

```suggestion
  kubeconfig_local_path = var.kubeconfig_local_path
```